src/kostentraeger/index.ts

```diff
@@ -0,0 +1,410 @@
+import { 
+    AbrechnungscodeEinzelschluessel as SGBVAbrechnungscodeEinzelschluessel, 
+    AbrechnungscodeGruppenschluessel, abrechnungscodeGruppenschluessel, 
+    getAbrechnungscodeEinzelschluessel
+} from "../sgb-v/codes";
+import { 
+    LeistungsartSchluessel as SGBXILeistungsartSchluessel
+} from "../sgb-xi/codes";
+import { LeistungserbringergruppeSchluessel } from "./edifact/codes";
+import { KassenartSchluessel } from "./filename/codes";
+import { 
+    CareProviderLocationSchluessel,
+    Institution,
+    InstitutionLink,
+    InstitutionList,
+    PaperDataType,
+    PapierannahmestelleLink
+} from "./types";
+
+
+/** Result of a KostentraegerIndex::find query */
+export type KostentraegerFindResult = {
+    /** Information of the institution with the IK given as pflegekasseIK parameter in the find function */
+    pflegekasse: Institution,
+    /** Information on the instutition which shall be the Kostenträger of this receipt */
+    kostentraeger: Institution,
+    /** Information on the institution for which the receipt shall be encrypted */
+    encryptTo?: Institution,
+    /** Information on the institution to which the receipt shall be sent */
+    sendTo: Institution 
+}
```

Comment:
This is the data that is returned in the end @michaelkamphausen 

I was not sure if you just need the IKs or also other info, so for now, all information about the institution is included.

Reply:
Looks good and pretty much complete to me.